lnrpc/invoicesrpc/addinvoice.go

```diff
@@ -47,6 +47,9 @@ type AddInvoiceConfig struct {
 	// channel graph.
 	ChanDB *channeldb.DB
 
+	// Graph holds a reference to the ChannelGraph database.
+	Graph *channeldb.ChannelGraph
```

Comment:
Nice catch!

Reply:
Oh my. Is this why people has been complaining about hop hints lately? 🤔